src/iceberg_extension.cpp

```diff
@@ -30,20 +30,14 @@ static unique_ptr<BaseSecret> CreateCatalogSecretFunction(ClientContext &context
 	for (const auto &named_param : input.options) {
 		auto lower_name = StringUtil::Lower(named_param.first);
 
-		if (lower_name == "key_id" || lower_name == "secret" || lower_name == "endpoint" ||
-		    lower_name == "aws_region" || lower_name == "oauth2_scope" || lower_name == "oauth2_server_uri") {
+		if (lower_name == "client_id" || lower_name == "client_secret" || lower_name == "endpoint" ||
+		    lower_name == "region" || lower_name == "oauth2_scope" || lower_name == "oauth2_server_uri") {
 			result->secret_map[lower_name] = named_param.second.ToString();
 		} else {
 			throw InternalException("Unknown named parameter passed to CreateIRCSecretFunction: " + lower_name);
 		}
 	}
 
-	// Get token from catalog
```

Comment:
This is moved to `IcebergCatalogAttach()` because `oauth2_server_uri` might not have been provided and the default is set there